TextToSpeech/TextToSpeechJsonRpc.cpp

```diff
@@ -21,12 +21,36 @@
 #include "UtilsJsonRpc.h"
 #include "UtilsUnused.h"
 #include "impl/TTSCommon.h"
+#include "UtilsisValidInt.h"
+#include <regex>
+#include "validator.h"
 
 #define GET_STR(map, key, def) ((map.HasLabel(key) && !map[key].String().empty() && map[key].String() != "null") ? map[key].String() : def)
+#define IS_INTEGER(string) (Utils::isValidUnsignedInt(const_cast<char*>(proxyVoice.c_str())))
 
 namespace WPEFramework {
 namespace Plugin {
 
+    struct InputValidation
```

Comment:
Please have this either in a separate header or as part of the validator header. And BTW, please rename validator.h to a suitable name which aligns with existing naming style.